rpm/0047-Force-recycling-of-gmp-droid-instances.-JB-51730.patch

```diff
@@ -0,0 +1,30 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
I guess this is moving 0046-sailfishos-gecko-Force-recycling-of-gmpdroid-instanc.patch from old-patches.